modules/social_features/social_event/modules/social_event_enrollment_max/social_event_max_enroll.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Event Max Enroll'
```

Comment:
The module is called `social_event_max_enroll` but is placed in the folder `social_event_enrollment_max`. This should be made consistent.